vms/platformvm/reward_validator_tx_test.go

```diff
@@ -195,202 +188,7 @@ func TestUnsignedRewardValidatorTxExecuteOnAbort(t *testing.T) {
-
-		vm.ctx.Lock.Unlock()
-	}()
-
-	initialSupply := vm.internalState.GetCurrentSupply()
-
-	vdrRewardAddress := ids.GenerateTestShortID()
-	delRewardAddress := ids.GenerateTestShortID()
-
-	vdrStartTime := uint64(defaultValidateStartTime.Unix()) + 1
-	vdrEndTime := uint64(defaultValidateStartTime.Add(2 * defaultMinStakingDuration).Unix())
-	vdrNodeID := ids.GenerateTestShortID()
-	vdrTx, err := vm.newAddValidatorTx(
-		vm.MinValidatorStake, // stakeAmt
-		vdrStartTime,
-		vdrEndTime,
-		vdrNodeID,        // node ID
-		vdrRewardAddress, // reward address
-		reward.PercentDenominator/4,
-		[]*crypto.PrivateKeySECP256K1R{keys[0]}, // fee payer
-		ids.ShortEmpty,                          // change addr
-	)
-	assert.NoError(err)
-
-	delStartTime := vdrStartTime
-	delEndTime := vdrEndTime
-	delTx, err := vm.newAddDelegatorTx(
-		vm.MinDelegatorStake, // stakeAmt
-		delStartTime,
-		delEndTime,
-		vdrNodeID,                               // node ID
-		delRewardAddress,                        // reward address
-		[]*crypto.PrivateKeySECP256K1R{keys[0]}, // fee payer
-		ids.ShortEmpty,                          // change addr
-	)
-	assert.NoError(err)
-
-	vm.internalState.AddCurrentStaker(vdrTx, 0)
-	vm.internalState.AddTx(vdrTx, status.Committed)
-	vm.internalState.AddCurrentStaker(delTx, 1000000)
-	vm.internalState.AddTx(delTx, status.Committed)
-	vm.internalState.SetTimestamp(time.Unix(int64(delEndTime), 0))
-	err = vm.internalState.Commit()
-	assert.NoError(err)
-	err = vm.internalState.(*internalStateImpl).loadCurrentValidators()
-	assert.NoError(err)
-
-	tx, err := vm.newRewardValidatorTx(delTx.ID())
-	assert.NoError(err)
-
-	_, onAbortState, err := tx.UnsignedTx.(UnsignedProposalTx).Execute(vm, vm.internalState, tx)
-	assert.NoError(err)
-
-	vdrDestSet := ids.ShortSet{}
-	vdrDestSet.Add(vdrRewardAddress)
-	delDestSet := ids.ShortSet{}
-	delDestSet.Add(delRewardAddress)
-
-	expectedReward := uint64(1000000)
-
-	oldVdrBalance, err := avax.GetBalance(vm.internalState, vdrDestSet)
-	assert.NoError(err)
-	oldDelBalance, err := avax.GetBalance(vm.internalState, delDestSet)
-	assert.NoError(err)
-
-	onAbortState.Apply(vm.internalState)
-	err = vm.internalState.Commit()
-	assert.NoError(err)
-
-	// If tx is aborted, delegator and delegatee shouldn't get reward
-	newVdrBalance, err := avax.GetBalance(vm.internalState, vdrDestSet)
-	assert.NoError(err)
-	vdrReward, err := math.Sub64(newVdrBalance, oldVdrBalance)
-	assert.NoError(err)
-	assert.Zero(vdrReward, "expected delegatee balance not to increase")
-
-	newDelBalance, err := avax.GetBalance(vm.internalState, delDestSet)
-	assert.NoError(err)
-	delReward, err := math.Sub64(newDelBalance, oldDelBalance)
-	assert.NoError(err)
-	assert.Zero(delReward, "expected delegator balance not to increase")
-
-	newSupply := vm.internalState.GetCurrentSupply()
-	assert.Equal(initialSupply-expectedReward, newSupply, "should have removed un-rewarded tokens from the potential supply")
-}
+// ?@jax again not a fan
```

Comment:
Pls. resolve / remove comment, I'm personally fine with removal, we need to add for sure deposit TestCases instead.
But they will be completely different to what we have here...